<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HealthTest</class>
 <widget class="QWidget" name="HealthTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>320</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>231</width>
     <height>227</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="Age">
        <property name="text">
         <string>年龄:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="age"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="Height">
        <property name="text">
         <string>身高(cm):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="height"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="Weight">
        <property name="text">
         <string>体重(kg):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="weight"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="S_pressure">
        <property name="text">
         <string>血压收缩压(mmHg):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="s_pressure"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="D_pressure">
        <property name="text">
         <string>血压舒张压(mmHg):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="d_pressure"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="BloodSugar">
        <property name="text">
         <string>血糖:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="blood_sugar"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="VC">
        <property name="text">
         <string>肺活量(ml):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="vc"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
